### PR TITLE
18069 remove ie8 support

### LINKED ISSUE #18069: Remove IE8 support
Finally 🎉 

- [ ] Remove IE8 from the karma test suite and CI setup
- [ ] Remove all VML related modules
- [ ] Remove all oldIE references and workarounds in the rest of the code

### PR DESCRIPTION
**Pull request by MarkusBarstad**
Removed support for IE 8 and older.

#### Upgrade notes
* The top-level `labels` option has been deprecated for a long time and is now removed. Use [Annotations](https://www.highcharts.com/docs/advanced-chart-features/annotations-module) instead.
* Time-related settings on the [global](https://api.highcharts.com/highcharts/global) options structure are removed. Use the [time](https://api.highcharts.com/highcharts/time) options instead.

### To do
- [x] Remove IE8 from the karma test suite and CI setup
- [x] Remove all VML and oldie related modules
- [x] Remove all oldIE references and workarounds in the rest of the code. Search for "IE8", "msie", "VML" and similar terms throughout the codebase, but do not remove stuff that is used by IE9-11.
- [x] Remove [SVGRenderer.isSVG](https://github.com/highcharts/highcharts/blob/0f8672da53093014fc5cb95d88c62a5be31eff17/ts/Core/Renderer/SVG/SVGRenderer.ts#L212). We don't need it anymore because it was always used for distinguishing the VMLRenderer.
- [x] Update the docs. Compatibility table, FAQ and possibly other mentions. Search for things like IE and VML.
 
Closes #18069.

### CODE REVIEW

**Review by TorsteinHonsi — COMMENTED** [review]
Some preliminary comments 👇

File: ts/Core/Renderer/SVG/SVGElement.ts [hunk 1]
```diff
@@ -531,7 +531,7 @@ class SVGElement implements SVGElementLike {
 
         // When the page is hidden save resources in the background by not
         // running animation at all (#9749).
-        if (pick(doc.hidden, doc.msHidden, doc.webkitHidden, false)) {
+        if (pick(doc.hidden, false)) {
```
[the remark this answers is not in the document]
Comment (MarkusBarstad) [reply ts/Core/Renderer/SVG/SVGElement.ts:534]: Yeah, good point. 

I can see that it [is supported from IE 10](https://caniuse.com/?search=document.hidden) so I guess we will have to keep at least msHidden to support IE9. 

I am having a hard time finding docs on support for webkitHidden but I will use a browser-tester and see if we need it.

[the remark this answers is not in the document]
Comment (MarkusBarstad) [reply ts/Core/Renderer/SVG/SVGElement.ts:534]: I am struggling a bit to find data on this (as well as sufficiently testing it) but `caniuse.com` [claims document.hidden](https://caniuse.com/?search=document.hidden) is not supported before IE10.

[the remark this answers is not in the document]
Comment (TorsteinHonsi) [reply ts/Core/Renderer/SVG/SVGElement.ts:534]: Okay, thanks! Let's just go with `if (doc.hidden)` then. If it becomes a problem (not likely), we can make a simple polyfill for the legacy builds.

File: ts/Core/Series/Series.ts [hunk 2]
```diff
@@ -3352,8 +3352,7 @@ class Series {
             hasRendered = series.hasRendered,
             chartSeriesGroup = chart.seriesGroup,
             inverted = chart.inverted;
-        // Animation doesn't work in IE8 quirks when the group div is
-        // hidden, and looks bad in other oldIE
+
         let animDuration = (!series.finishedAnimating && chart.renderer.isSVG) ?
```
Comment (TorsteinHonsi) [inline ts/Core/Series/Series.ts:3356]: The `chart.renderer.isSVG` condition can be removed here, ref the removed comment.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks, nice work! See inline comments.

**Review by TorsteinHonsi — APPROVED** [review]
🎉

**Pull request merged**